README.md

```diff
@@ -3,16 +3,19 @@
 [![Build Status](https://travis-ci.org/rhysd/neovim-component.svg?branch=master)](https://travis-ci.org/rhysd/neovim-component)
 
 
-This component provides `<neovim-editor>` HTML custom element built on [Polymer](https://github.com/Polymer/polymer) and [flux](https://github.com/facebook/flux).
-It provides a frontend of [Neovim editor](https://github.com/neovim/neovim) using Neovim's MessagePack API and you can embed Neovim editor easily to your application.
+This component provides `<neovim-editor>`, an HTML custom element built on [Polymer](https://github.com/Polymer/polymer) and [flux](https://github.com/facebook/flux).
+It provides a frontend for the [Neovim editor](https://github.com/neovim/neovim) using Neovim's MessagePack API. It allows you to easily embed a Neovim-backed editor
+into your application.
```

Comment:
It's markdown, so it displays the same either way, but I can't still remove it if you like. Convention here is usually to format the whole document to 80 characters wide, but it doesn't matter particularly to me.